examples/resources/awscc_amplify_domain/amplify_domain.tf

```diff
@@ -0,0 +1,35 @@
+resource "awscc_amplify_app" "example" {
+  name = "app"
+
+  # Setup redirect from https://example.com to https://www.example.com
+  custom_rules = [
```

Comment:
Suggestion here based on personal experience, I would rather to have another example for redirecting from apex domain to sub domain like www.example.com

Some customers required to use their apex domain as their company default domain.
For the basic amplify_domain, probably you can use a custom_rules for redirecting to a HTTP status code 404 page.